windows/check_quiesce_snapshot/check_vmx_disk_enable_uuid.yml

```diff
@@ -9,10 +9,28 @@
 
 # Get VM vmx config
 - include_tasks: ../../common/vm_get_extra_config.yml
+
+# Known issue on Windows Server 2022 guest ID on ESXi 7.0U3c build, fixed on ESXi 7.0.3P04, KB article will be published
```

Comment:
Is ESXi 7.0.3 P04 same as ESXi 7.0 Update 3d? If yes, I suggest using ESXi 7.0 Update 3d as it can be easily found from https://kb.vmware.com/s/article/2143832.  The other looks good to me.

Reply:
Yeah, it's the same release. Thanks.